book/06-github/sections/2-contributing.asc

```diff
@@ -407,12 +406,12 @@ There are actually quite a number of web services that make use of emoji charact
 http://www.emoji-cheat-sheet.com
 ====
 
-====== Images
+===== Images
 
 This isn't technically GitHub Flavored Markdown, but it is incredibly useful. In addition to adding Markdown image links to comments, which can be difficult to find and embed URLs for, GitHub allows you to drag and drop images into text areas to embed them.
 
 [[_md_drag]]
 .Drag and drop images to upload them and auto-embed them.
 image::images/markdown-08-drag-drop.png[Drag and drop images]
 
-If you look back at <<_pr_references>>, you can see a small ``Parsed as Markdown'' hint above the text area. Clicking on that will give you a full cheat sheet of everything you can do with Markdown on GitHub.
+If you at <<_md_drag>>, you can see a small ``Parsed as Markdown'' hint above the text area. Clicking on that will give you a full cheat sheet of everything you can do with Markdown on GitHub.
```

Comment:
A verb is missing in the first part of this sentence, the one you removed (look back).